### PR TITLE
First batch edits

### PR DESCRIPTION
**Pull request by kaylaewood**
Edits on `vampire`, `dragon`, `hobbit`, and `pirate`

### CODE REVIEW

**Comment by kaylaewood** [comment]
vampire: add drink function with lots of conditionals

dragon: remove color (wasn't used), add interpolation function

hobbit: change to passing an object, add return statements to `getRing` 

pirate: add functionality around `cursed` property (`buyBlessing` and `robShip`), add spice to functions, new function of `buyBlessing`

**Comment by kaylaewood** [comment]
@hannahhch @sertmer given the changes I've made, I'm going to switch the recommended order in the README to have it be dragon -> vampire. Let me know if you have any concerns about that!

**Comment by sertmer** [comment]
Can't wait to give the students these new versions!

**Review by sertmer — APPROVED** [review]

File: mythical-creatures/test/pirate-test.js [hunk 1]
```diff
@@ -55,7 +51,7 @@ describe('Pirate', function() {
   });
 
   it.skip('should get 100 gold pieces when robbing a ship', function() {
-    var pirate = new Pirate('Robbie');
+    var pirate = new Pirate('Hannah');
```
Comment (hannahhch) [inline mythical-creatures/test/pirate-test.js:54]: Excellent change

**Review by hannahhch — APPROVED** [review]

**Pull request merged**